### ISSUE TITLE
Should have way to start menuItems with subitems expanded

### ISSUE THREAD

**Issue by wch** [rstudio/shinydashboard#38]

**Comment by shrektan** [comment]
I want to add **should have way to start box with box collapsed**. Is this possible in the current code?

**Comment by wch** [comment]
@shrektan See #33.

**Comment by wch** [comment]
It looks like this might require replacing the ` $.AdminLTE.tree` function from app.js.

**Comment by shrektan** [comment]
Good news! Thanks!

**Comment by pssguy** [comment]
It would also be useful to show  and highlight the newly active menuItem or menuSubItem when using 
updateTabItems

**Comment by wch** [comment]
The menuItem expanded/collapsed state should also be bookmarkable.

**Comment by bborgesr** [comment]
@wch I guess this means that we should also allow bookmarking of the expanded/collapsed state of the whole sidebar, right? (this may affect #186)

**Comment by bborgesr** [comment]
**Note for future stumblers on this issue**:

Once #199 is merged, the way to start with a "childfull" `menuItem()` expanded will to use the `startExpanded` argument. For example, in the app below, the last `menuItem` (`"Charts"`) starts expanded and it's first child (`"Sub-item 1"`) starts selected:

```r
shinyApp(
  ui = dashboardPage(
    dashboardHeader(),
    dashboardSidebar(
      sidebarMenu(
        # Setting id makes input$tabs give the tabName of currently-selected tab
        id = "tabs",
        menuItem("Dashboard", tabName = "dashboard", icon = icon("dashboard")),
        menuItem("Widgets", icon = icon("th"), tabName = "widgets", badgeLabel = "new"),
        menuItem("Charts", icon = icon("bar-chart-o"), startExpanded = TRUE,
          menuSubItem("Sub-item 1", tabName = "subitem1", selected = TRUE),
          menuSubItem("Sub-item 2", tabName = "subitem2")
        )
      )
    ),
    dashboardBody(),
    title = "Dashboard example"
  ),
  server = function(input, output) { }
)
```

**Comment by bborgesr** [comment]
Closed in #199
